packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -1,12 +1,38 @@
 import { declare } from "@babel/helper-plugin-utils";
 import { addDefault, isModule } from "@babel/helper-module-imports";
 import { types as t } from "@babel/core";
+import getTargets, {
+  filterItems,
+  isRequired,
+} from "@babel/helper-compilation-targets";
+import corejs2Polyfills from "@babel/compat-data/corejs2-built-ins";
+import corejs3Polyfills from "core-js-compat/data";
 
 import getCoreJS2Definitions from "./runtime-corejs2-definitions";
 import getCoreJS3Definitions from "./runtime-corejs3-definitions";
 import { typeAnnotationToString } from "./helpers";
 import getRuntimePath from "./get-runtime-path";
 
+// corejs sometimes lumps multiple functions/identifiers under one name
+corejs3Polyfills["web.set-immediate"] = corejs3Polyfills["web.immediate"];
+corejs3Polyfills["web.clear-immediate"] = corejs3Polyfills["web.immediate"];
+corejs3Polyfills["web.set-interval"] = corejs3Polyfills["web.timers"];
+corejs3Polyfills["web.clear-interval"] = corejs3Polyfills["web.timers"];
+corejs3Polyfills["web.set-timeout"] = corejs3Polyfills["web.timers"];
+corejs3Polyfills["web.clear-timeout"] = corejs3Polyfills["web.timers"];
+corejs3Polyfills["es.array.entries"] = corejs3Polyfills["es.array.iterator"];
+corejs3Polyfills["es.array.keys"] = corejs3Polyfills["es.array.iterator"];
+corejs3Polyfills["es.array.values"] = corejs3Polyfills["es.array.iterator"];
+corejs3Polyfills["es.string.trim-left"] =
+  corejs3Polyfills["es.string.trim-start"];
+corejs3Polyfills["es.string.trim-right"] =
+  corejs3Polyfills["es.string.trim-end"];
+corejs3Polyfills["es.symbol.for"] = corejs3Polyfills["es.symbol"];
+corejs3Polyfills["es.symbol.key-for"] = corejs3Polyfills["es.symbol"];
+corejs3Polyfills["es.object.get-own-property-symbols"] =
+  corejs3Polyfills["es.symbol"];
+corejs2Polyfills["es6.symbol.iterator"] = corejs2Polyfills["es6.symbol"];
```

Comment:
We can add a new `es6.symbol.iterator` item in

https://github.com/babel/babel/blob/2f31ecf85d85cb100fa08d4d9a09de0fe4a117e4/packages/babel-compat-data/scripts/data/corejs2-built-in-features.js#L294

IMO in old browsers `es6.symbol.iterator` should have more support than the general `es6.symbol` feature as it does not depend on the other well known symbols.

Reply:
Just to make sure I fully understand that file's format, would I add something like the following?
```json
"es6.symbol.iterator": "well-known symbols / Symbol.iterator"
```

Reply:
Done.

Reply:
Based on the failed tests, was I supposed to take `"well-known symbols / Symbol.iterator"` out of `"es6.symbol"`? And then regenerate the snapshots for the tests?

Reply:
I actually don't think I can make this change. Making this change causes `@babel/preset-env` to emit `require("core-js/modules/es6.symbol.iterator");` statements, but [`core-js@2` doesn't have a module with that filename](https://github.com/zloirock/core-js/tree/v2/modules).

We might be stuck doing `corejs2Polyfills["es6.symbol.iterator"] = corejs2Polyfills["es6.symbol"];`.

Reply:
@JLHwung just tagging in case you didn't get a notification about this. Let me know what your thoughts are!

Reply:
Good catch. Why we need `corejs2Polyfills["es6.symbol.iterator"] = ...` if it turns out core-js@2 does not have one?

Reply:
It's used to decide whether to replace `something[Symbol.iterator]()` calls with the `getIterator(something)` helper, `Symbol.iterator in something` usages with the `isIterable(something)` helper, and `something[Symbol.iterator]` usages with the `getIteratorMethod(something)` helper. Look for `polyfillPaths.has("symbol/iterator")` in the file to see what I mean.

These helper methods are available in the [`fn` directory of `core-js@2`](https://github.com/zloirock/core-js/tree/v2/fn).

Reply:
@JLHwung went ahead and reverted the changes.